{{cookiecutter.project_name}}/environments/ceph/configuration.yml

```diff
@@ -25,3 +25,23 @@ ceph_conf_overrides:
     osd pool default size: 3
   mon:
     mon allow pool delete: true
+{% raw -%}
+  "client.rgw.{{ hostvars[inventory_hostname]['ansible_hostname'] }}.rgw0":
+    "rgw content length compat": "true"
+    "rgw enable apis": "swift, s3, admin"
+    "rgw keystone accepted admin roles": "admin"
+    "rgw keystone accepted roles": "member, admin"
+    "rgw keystone admin domain": "default"
+    "rgw keystone admin password": "{{ ceph_rgw_keystone_password }}"
+    "rgw keystone admin project": "service"
+    "rgw keystone admin tenant": "service"
+    "rgw keystone admin user": "ceph_rgw"
+    "rgw keystone api version": "3"
+    "rgw keystone implicit tenants": "true"
+{%- endraw %}
+    "rgw keystone url": "http://{{cookiecutter.fqdn_internal}}:5000"
+    "rgw keystone verify ssl": "false"
+    "rgw s3 auth use keystone": "true"
+    "rgw swift account in url": "true"
+    "rgw swift versioning enabled": "true"
+    "rgw verify ssl": "false"
```

Comment:
Any chance to make this configurable? We use official certificates

Reply:
I would see that as post processing. Like the http in rgw keystone url. Otherwise we have to set a lot of parameters, which IMO is not the purpose of the cookie cutter.